main.py

```diff
@@ -160,18 +215,21 @@ def transfer(self, recipient_account: Account, amount: float) -> None:
 customer1 = bank.create_account("Normal", 1000)
 customer2 = bank.create_account("Normal", 500)
 customer3 = bank.create_account("Debit", 1500)
+customer4 = bank.create_account("Savings", 1500)
```

Comment:
This line throws an error.
Looks like in `create_account` method we don't have `Savings` as a `account_type`. 

We should add another condition as following:
```
 elif account_type == "Savings":
            account = SavingsAccount(account_number, initial_balance)
```